--- a/write_font.py
+++ b/write_font.py
@@ -730,3 +730,3 @@ def _generate_color_font(config: FontConfig, inputs: Iterable[InputGlyph]):

-        color_glyphs.append(
+        try: color_glyphs.append(
             ColorGlyph.create(
@@ -743,2 +743,3 @@ def _generate_color_font(config: FontConfig, inputs: Iterable[InputGlyph]):
         )
+        except: print(f'Glyph "{glyph_input.glyph_name}" excluded due to compatibility issues with fonttools.')
     color_glyphs = tuple(color_glyphs)

--- a/ninja.py
+++ b/ninja.py
@@ -122,1 +122,1 @@
-    ninja_cmd = ["ninja", "-C", os.path.dirname(build_file)]
+    ninja_cmd = ["ninja", "-k", "0", "-C", os.path.dirname(build_file)]
